src/libs/ActiveClientManager/index.ts

```diff
@@ -54,18 +53,24 @@ Onyx.connect({
  * We want to ensure we have no duplicates and that the activeClient gets added at the end of the array (see isClientTheLeader)
  */
 function init() {
-    activeClients = _.without(activeClients, clientID);
+    activeClients = activeClients.filter((id) => id !== clientID);
     activeClients.push(clientID);
     ActiveClients.setActiveClients(activeClients).then(resolveSavedSelfPromise);
 }
 
 /**
  * The last GUID is the most recent GUID, so that should be the leader
- *
- * @returns {Boolean}
  */
-function isClientTheLeader() {
-    return _.last(activeClients) === clientID;
+function isClientTheLeader(): boolean {
```

Comment:
This case is up to you, however TS sets here automatically as well.
```suggestion
function isClientTheLeader() {
```

Reply:
@rezkiy37 We should leave return types explicit for most cases, unless the function is like one-line and the return type is `void`.